src/components/keyboardnavigation.js

```diff
@@ -1,36 +1,124 @@
-define(['inputManager', 'focusManager'], function(inputManager, focusManager) {
-    'use strict';
+define(["inputManager", "layoutManager"], function (inputManager, layoutManager) {
+    "use strict";
 
     console.log("keyboardnavigation");
 
+    /**
+     * Key name mapping.
+     */
+    // Add more to support old browsers
+    var KeyNames = {
+        13: "Enter",
+        19: "Pause",
+        27: "Escape",
+        32: "Space",
+        37: "ArrowLeft",
+        38: "ArrowUp",
+        39: "ArrowRight",
+        40: "ArrowDown",
+        412: "MediaRewind", // MediaRewind (Tizen/WebOS)
+        413: "MediaStop", // MediaStop (Tizen/WebOS)
+        415: "MediaPlay", // MediaPlay (Tizen/WebOS)
+        417: "MediaFastForward", // MediaFastForward (Tizen/WebOS)
+        461: "Back", // Back (WebOS)
+        10009: "Back", // Back (Tizen)
+        10232: "MediaTrackPrevious", // MediaTrackPrevious (Tizen)
+        10233: "MediaTrackNext", // MediaTrackNext (Tizen)
+        10252: "MediaPlayPause" // MediaPlayPause (Tizen)
```

Comment:
Maybe put the comments on the line above each value so we can expand on them if we need to.